examples/graphql-server-typeorm-postgres/src/index.ts

```diff
@@ -0,0 +1,84 @@
+import { ApolloServer, makeExecutableSchema } from 'apollo-server';
+import { mergeTypeDefs, mergeResolvers } from 'graphql-toolkit';
+
+import { AccountsModule } from '@accounts/graphql-api';
+import { AccountsPassword } from '@accounts/password';
+import { AccountsServer } from '@accounts/server';
+import { AccountsTypeorm } from '@accounts/typeorm';
+import { connect } from './connect';
+
+export const createAccounts = async () => {
+  const connection = await connect(process.env.TEST_DATABASE_URL);
+  // Like, fix this man!
+  const tokenSecret = 'ACCOUNTS_SECRET' || 'not very secret secret';
```

Comment:
this looks a bit weird :)

Reply:
Oh sorry, I totally misspelled that since there's no TEST_DB_URL, 

```typescript

export const createAccounts = async () => {
  const connection = await connect(process.env.DATABASE_URL);
  // Like, fix this man!
  const tokenSecret = 'process.env.ACCOUNTS_SECRET' || 'change this in .env';
```

And probably add an .env.example
```bash
DATABASE_URL=postgres://postgres@localhost:5432/postgres
ACCOUNTS_SECRET=Change_me_please
```
Would that be OK?

Reply:
OK, done, I shouldn't add tests right, I didn't see them in the rest of the examples, it's enough with making yarn install and core-packages tests pass on CircleCI as it is.